### ISSUE TITLE
Offer ability to visually distinguish/filter different types of locations on maps

### ISSUE THREAD

**Issue by joeg8612** [massgov/feature-requests#93]
As a content author, I would like to be able to visually distinguish/filter different types of locations on a map within a list set.  Sometimes there are locations that provide different types of services.  I want visitors to be able to filter based on those characteristics.

**Reference**
JIRA DP-5143

**Comment by alarrimore** [comment]
This looks like the same issues at https://github.com/massgov/feature-requests/issues/129

**Comment by brookekaranovich** [comment]
I agree with this request. This is something that DTA is finding a need to be able to do as we have different levels of access at different locations. This is especially important as agencies begin to work with others to expand footprints across the state. For example, clients can access a certain level of support at a community partner, while they can get full service at a DTA office. This is similar to RMV and AAA. We need this to be very clear though, which is where a filter would come in.
